### PR TITLE
Use `getResponseJson` in `ApiDocsView` so that API Docs have full API response

### PR DESCRIPTION
**Pull request by michaelzetune**
### Description

Before this PR, the API docs page only shows part of the API response, namely the part under `"payload"`. Technically, this is enough, since the only other part of the the response is the `nextPageToken`. However, in the name of completeness, we want to show the full response so that API consumers understand what they are getting back from CiviForm.

This PR uses *the same code as the API itself* to generate this preview, so there will not be divergence between them. As an added bonus, the screenshot tests will operate as an implicit "golden test" in case the API were to change in a way we didn't notice.

<details>
<summary>Before</summary>

image
</details>

<details>
<summary>After</summary>

image

</details>

## Release notes

Improves the API Response Preview on API docs.

#### General

Read the full guidelines for PRs [here](https://docs.civiform.us/contributor-guide/developer-guide/technical-contribution-guide#guidelines-for-pull-requests)

- [x] Added the correct label: < feature | bug | dependencies | infrastructure | ignore-for-release | database >
- [x] Created unit and/or browser tests which fail without the change (if possible)
- [x] Performed manual testing (Chrome and Firefox if it includes front-end changes)
- [ ] Extended the README / documentation, if necessary

#### User visible changes

- [ ] Followed steps to [internationalize new strings](https://docs.civiform.us/contributor-guide/developer-guide/internationalization-i18n#internationalization-for-application-strings)
- [ ] Wrote browser tests using the [validateAccessibility](https://sourcegraph.com/github.com/civiform/civiform/-/blob/browser-test/src/support/index.ts?L437:14&subtree=true) method
- [ ] Tested on mobile view. See [mobile device mode](https://developer.chrome.com/docs/devtools/device-mode/)
- [ ] Manually tested at 200% size
- [ ] Manually evaluated tab order

### Issue(s) this completes

Relates to #5262.

### CODE REVIEW

File: browser-test/image_snapshots/api_docs_test/api-docs-page-accordion-open.png
Comment (michaelzetune) [inline browser-test/image_snapshots/api_docs_test/api-docs-page-accordion-open.png:1]: Before, we would only show everything inside `payload` in the gray box. Now, we show the entire response.

File: server/app/controllers/api/CiviFormApiController.java [hunk 1]
```diff
@@ -53,31 +49,4 @@ protected void assertHasProgramReadPermission(Http.Request request, String progr
       throw new UnauthorizedApiRequestException(apiKey, programSlug);
     }
   }
-
```
Comment (michaelzetune) [inline server/app/controllers/api/CiviFormApiController.java:56]: Moved to `JsonExporter` (which makes a lot more sense anyways)

File: server/app/services/CfJsonDocumentContext.java [hunk 2]
```diff
@@ -600,18 +597,6 @@ public String asJsonString() {
     return jsonData.jsonString();
   }
 
-  public String asPrettyJsonString() {
```
Comment (michaelzetune) [inline server/app/services/CfJsonDocumentContext.java:603]: Moved to `JsonPrettifier`

File: server/app/services/export/JsonExporter.java [hunk 3]
```diff
@@ -147,6 +155,33 @@ CfJsonDocumentContext buildJsonApplication(JsonExportData jsonExportData) {
     return jsonApplication;
   }
 
+  public String getResponseJson(
```
Comment (michaelzetune) [inline server/app/services/export/JsonExporter.java:158]: Unmodified from `CiviFormApiController`

File: server/app/services/export/JsonPrettifier.java [hunk 4]
```diff
@@ -0,0 +1,20 @@
+package services.export;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
+public class JsonPrettifier {
+  public static String asPrettyJsonString(String original) {
```
Comment (michaelzetune) [inline server/app/services/export/JsonPrettifier.java:8]: Moved and adapted from`CfJsonDocumentContext`

File: server/app/services/export/JsonPrettifier.java [hunk 5]
```diff
@@ -0,0 +1,20 @@
+package services.export;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+
```
Comment (dkatzz) [inline server/app/services/export/JsonPrettifier.java:6]: **suggestion:** Can you add a javadoc

**Review by dkatzz — APPROVED** [review]
Nice!

File: server/app/services/export/JsonPrettifier.java (hunk 5 above)
Comment (michaelzetune) [reply server/app/services/export/JsonPrettifier.java:6]: Done.

**Pull request merged**